packages/workbox-core/test/bundle/node/controllers/test-WorkboxCore.js

```diff
@@ -0,0 +1,34 @@
+import chai from 'chai';
```

Comment:
Huh, that's weird. I assumed that it would work by default. Not sure if it was worth the extra effort, then, but thanks for addressing.